### PR TITLE
remove Text module

### PR DESCRIPTION
**Pull request by bergey**
What do folks think about just removing this?  Explanation below.

Since diagrams-cairo switched to Pango for text layout in b953ae90d3c1ba13c15bc5a86c5fa4851da16ce7, the functions in
Cairo.Text have been obsolete.  They don't use the same code path as
actual rendering, so in general, the sizes they report are wrong.  The
module has always been marked unsupported, so removing it is easiest,
and avoids a likely source of confusion.

An alternative to removing it would be updating it to use Pango.
However, Pango differs substantially.  I'd want to use the FontMetrics
type that (Haskell) pango provides, and the font metrics depend on the
final size, so all the functions would need more parameters --
basically, a complete redesign of the module.  Since this would be
Cairo-specific, and since more general solutions have been proposed, I'd
rather not do this unless there's a clear user need.

### CODE REVIEW

**Comment by jeffreyrosenbluth** [comment]
I think we should remove it

**Comment by fryguybob** [comment]
+1 remove

**Comment by byorgey** [comment]
Agreed, kill it.

**Comment by bergey** [comment]
OK.  Merging now.  I'll fix travis later, since the failure looks unrelated.
